src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -2,11 +2,19 @@
 
 public class BinaryString {
 
-    /**
-     * Напишите метод toBinaryString(), который принимает на вход целое число value,
-     * а возвращает String с представлением этого числа в двоичном виде.
-     */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+        String result = "";
```

Comment:
can you create the solution without `result` variable?

Reply:
done